integration/contracts/SpecAndRunLog.sol

```diff
@@ -18,12 +18,14 @@ contract SpecAndRunLog is Chainlinked, Ownable {
   }
 
   function request() public {
-    string[] memory tasks = new string[](1);
-    tasks[0] = "httppost";
+    string[] memory tasks = new string[](2);
+    tasks[0] = "sleep";
+    tasks[1] = "httppost";
 
     ChainlinkLib.Spec memory spec = newSpec(tasks, this, "fulfill(bytes32,bytes32)");
     spec.add("msg", "hello_chainlink");
     spec.add("url", "http://localhost:6690");
+    spec.add("duration", "3s");
```

Comment:
We might want to support `spec.addUint("duration", 3);` here. I'm thinking of scenarios where the contract would compute the difference between a future block timestamp and the current, and use that for the given input here.